### PR TITLE
docs: README.md

### PR DESCRIPTION
**Pull request by Azleal**
add installation command

### Description

the README is lack of installation commands, so I added them.

### Rationale

for users who don't know how to install this tool, it would be better if installation commands are provided.

### Changes

Notable changes:
* README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,16 @@ for any bug bounty. We advise you to be careful and experiment on the network at
 
 ## Cmd usage
 
+### installation
+
+```
+git clone https://github.com/bnb-chain/greenfield-cmd.git
+cd greenfield-cmd
+make build
```
Comment (flywukong) [inline README.md:19]: make PR to develop branch instead of master

Comment (Azleal) [reply README.md:19]: done

**Review by flywukong — APPROVED** [review]

**Review by flywukong — COMMENTED** [review]
please pull the latest develop branch and add the commit on it , then push again

**Comment by Azleal** [comment]
> please pull the latest develop branch and add the commit on it , then push again

I rebased from the latest develop branch. but I don't think this is necessary, because there's no conflict. I don't quite get why it should be done though.

**Pull request merged**